docs/install/linux_source.rst

```diff
@@ -12,8 +12,9 @@ PyNE has the following dependencies:
    #. `Fortran compiler <https://gcc.gnu.org/wiki/GFortran>`_
    #. `C++ compiler <https://gcc.gnu.org/>`_
    #. `CMake <http://www.cmake.org/>`_ (>= 2.8.5)
-   #. `NumPy <http://www.numpy.org/>`_ (>= 1.8.0)
+   #. `NumPy <http://www.numpy.org/>`_ (1.11.0)
    #. `SciPy <http://www.scipy.org/>`_
+   #. `Jinja2 <http://jinja.pocoo.org/docs/2.9/intro/>`_
```

Comment:
Actually some tests were giving errors before jinja2 installation
I am not sure if it is optional though. Should I add it in optional dependencies?